src/Control/Optics/Linear/Internal.hs

```diff
@@ -44,71 +455,157 @@ import Data.Void
 import Prelude.Linear
 import qualified Prelude as P
 
+
+-- # Data Type Definitions
+-------------------------------------------------------------------------------
+
+-- | A wrapper for an Optic that uses a "Data.Profunctor.Linear" @arr@
```

Comment:
This line sounds like it's specialisation of a more generic data type, and my instinct when I read "a wrapper" is just skipping it and looking for the actual type, which doesn't exist in this case. I'd probably just paste the source as the haddock.